fastlane/Appfile

```diff
@@ -1,3 +1,3 @@
 app_identifier "io.getstream.ChatExample" # The bundle identifier of your app
-apple_dev_portal_id "bahadir.onc@icloud.com"  # Apple Developer Account
+apple_dev_portal_id "bahadir.onc@icloud.com" # Apple Developer Account
```

Comment:
This is a bit concerning, shouldn't we update this? 😄

Reply:
🙈